tests/test_addsubdir.py

```diff
@@ -0,0 +1,61 @@
+""" 
+test_addsubdir.py --- test if build_component.py can copy files to subdirectories 
+of the build target dir.
+
+The scripts can call build_component.py directly not initialize.py, so you should 
+execute initialize.py before running it.
```

Comment:
I had to re-read this a few times, but I think it's saying that `build.py` should be the only build script that calls `build_component.py`, and the `initialize.py` build script should be manually executed by the user before running `test_addsubdir.py`.  Is that correct?